### DISCUSSION TITLE
how to deal with the LLM download issue

### DISCUSSION THREAD

**Discussion by candy18** [janhq/jan#4367] [Get Help] [answered]
Download stopped at 53.17% with no error messages, despite the internet working fine. The version of Jan is v0.5.12

How can I resolve this and complete the download?

![CleanShot 2024-12-30 at 23 49 45@2x](https://github.com/user-attachments/assets/e4f30de2-32ea-492b-a8e1-6f0deb55ab6e)

**Comment by louis-jan** [comment, marked as answer]
The model downloader has been improved over previous versions. Please try the latest version.
